---
title: 'DahShu Webinar: Statistical Analysis of Single-Cell RNA-seq Data with Multiple Samples'
author: ''
date: '2022-01-21T09:00:00-07:00'
slug: dahshu-webinar-statistical-analysis-of-single-cell-rna-seq-data-with-multiple-samples
categories: []
tags: []
type: webinar
url_register: ~
url_freeregister: https://dahshu.wildapricot.org/event-4644115
url_slides: ~
url_video: no
url_agenda: ~
url_website: ~
url_audio: ~
url_code: ~
url_pdf: ~
date_end: '2022-01-21T10:00:00-07:00'
all_day: no
publishDate: '2022-01-21T09:43:02-08:00'
authors: []
featured: no
image:
  caption: ''
  focal_point: ''
  preview_only: no
slides: ''
projects: []
location: ~
address:
  street: ~
  city: ~
  region: ~
  postcode: ~
  country: ~
summary: ~
abstract: ~
speaker: Hongkai Ji (Johns Hopkins Bloomberg School of Public Health)
---
<!--more-->
As single-cell RNA-seq (scRNA-seq) is increasingly used in biomedical research, scRNA-seq datasets with multiple patient samples become common. In this talk, I will introduce our recent research on analyzing multi-sample scRNA-seq data. I will introduce TreeCorTreat, an open source R package that uses a tree-based correlation screen to help explore and visualize the association between samplesâ€™ phenotype and their transcriptomic features at multiple cell type resolutions. I will also introduce Lamian, a statistical framework for differential pseudotime analysis with multiple samples. I will demonstrate these tools using both simulation and real data including an integrative analysis of COVID-19 single-cell RNA-seq data.