---
# Documentation: https://wowchemy.com/docs/managing-content/
type: course
title: Polygenic Risk Prediction and Equitable Disease Prevention
url_register: https://www.amstat.org/ASA/Education/Web-Based-Lectures.aspx#PRPEDP
url_freeregister: 
url_slides:
url_video: false
url_agenda:
url_website:
url_audio:
url_code:
url_pdf:

# Talk start and end times.
#   End time can optionally be hidden by prefixing the line with `#`.
date: 2021-04-22T11:00:00-07:00
date_end: 2021-04-22T13:00:00-07:00
all_day: false

# Schedule page publish date (NOT event date).
publishDate: 2021-03-28T21:15:57-07:00

authors: []
tags: []

# Is this a featured event? (true/false)
featured: false

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder. 
# Focal points: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight.
image:
  caption: ""
  focal_point: ""
  preview_only: false

# Custom links (optional).
#   Uncomment and edit lines below to show custom links.
# links:
# - name: Follow
#   url: https://twitter.com
#   icon_pack: fab
#   icon: twitter

# Markdown Slides (optional).
#   Associate this event with Markdown slides.
#   Simply enter your slide deck's filename without extension.
#   E.g. `slides = "example-slides"` references `content/slides/example-slides.md`.
#   Otherwise, set `slides = ""`.
slides: ""

# Projects (optional).
#   Associate this post with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `projects = ["internal-project"]` references `content/project/deep-learning/index.md`.
#   Otherwise, set `projects = []`.
projects: []

location:
address:
  street:
  city:
  region:
  postcode:
  country:
summary:
abstract:
speaker: Nilanjan Chatterjee (Johns Hopkins University)
---
<!--more-->
Recent discoveries from large scale genome-wide association studies (GWAS) have raised the prospect of using polygenic risk scores in routine health care setting for the prediction of future incidence of large variety of complex diseases. However, as GWAS studies to date have been heavily biased towards European origin populations, current polygenic risk scores often underperform in non-European populations and thus use of them can further exacerbate healthcare inequality. In this talk, I will review simple and advanced statistical methods for generating polygenic risk score using high-dimensional SNP data and describe theoretical characterizations of their expected performance, both in the population that underlies original studies and in a different population that is expected to have different distribution of allele frequencies and linkage disequilibrium (SNP-correlation). I will further describe novel Bayesian and machine learning based methods for building polygenic risk scores that can borrow information across GWAS studies of different ethnic groups, and thus makes best use of available data to generate more powerful polygenic risk scores across different ethnic groups. I will demonstrate potential utility for PRS in precision medicine using our recent studies on breast cancer.