---
draft: yes
title: Identification of Putative Causal Loci in Whole-Genome Sequencing Data via
  Knockoff Statistics
author: ''
date: '2021-03-12T16:00:00-08:00'
slug: identification-of-putative-causal-loci-in-whole-genome-sequencing-data-via-knockoff-statistics
categories:
  - Statistics
tags:
  - SFASA
type: webinar
url_register: ~
url_freeregister: https://docs.google.com/forms/d/e/1FAIpQLScmO00DOxCQSihrU3lqttCgjCsEeMLjvApxmvxBU5WXpsT_wA/viewform
url_slides: ~
url_video: false
url_audio: ~
url_code: ~
url_pdf: ~
noreplay: true
date_end: '2021-03-12T17:00:00-08:00'
all_day: no
publishDate: 
authors: []
featured: no
image:
  caption: ''
  focal_point: ''
  preview_only: no
slides: ''
projects: []
location: ~
address:
  street: ~
  city: ~
  region: ~
  postcode: ~
  country: ~
summary: ~
abstract: ~
speaker: Zihuai He (Stanford University)
---
<!--more-->
The analysis of whole-genome sequencing studies is challenging due to the large number of rare variants in noncoding regions and the lack of natural units for testing. Existing association tests often suffer from insufficient power due to the substantially increased burden of multiple testing and correspondingly stringent multiple testing corrections that do not appropriately account for correlations. Using population-level whole-genome sequencing data, we propose a new statistical method to detect and localize rare and common risk variants based on a recently developed knockoff framework. The proposed approach includes a novel sequential knockoff generator for both rare and common variants in whole-genome sequencing studies, paired with a powerful screening method to comprehensively analyze rare and common variants across the genome. The proposed knockoff generator is scalable to large whole-genome sequencing datasets, and the screening method is powerful in detecting signals across the genome with guaranteed false discovery rate control. Furthermore, it can (1) prioritize causal variants over associations due to linkage disequilibrium thereby improving interpretability; (2) help distinguish the signal due to rare variants from shadow effects of significant common variants nearby; (3) integrate multiple knockoffs for improved power, stability and reproducibility; and (4) flexibly incorporate state-of-the-art and future association tests to achieve the benefits proposed here. In applications to whole-genome sequencing data from the Alzheimer’s Disease Sequencing Project (ADSP) and COPDGene samples from NHLBI Trans-Omics for Precision Medicine (TOPMed) Program we show that our method compared with conventional association tests can lead to substantially more discoveries, including several new ones and many previously implicated in GWAS studies on Alzheimer’s Disease and lung function phenotypes with much larger sample sizes. 
